CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Change Log
 
+## [2.0.120230124]
+* CLI version update that provides: Devhost agent port change for compatibility with GRPC unofficial port and JSon serialization changes (STJ changes).
```

Comment:
💡☕️ 3 minor things:

- Empty line between header the thing getting released.
- Do we have reference of any official doc for the GRPC UnOfficial Port - if so please link it.
- Also might be add alias of folks who really contributed for this change. (suggest a suggestion) :) 

Thank you ❤️🙏

Reply:
Thanks @Tatsinnit! I updated the description to point to the port change PR which has the details you asked for which will keep this change long concise. Also added the space. Regarding the adding alias specifically, I rather keeping it generic to avoid missing someone.